grails-app/views/consumption/_filters.gsp

```diff
@@ -131,12 +131,12 @@
                         <g:hasRoleFinance>
                             <g:set var="hasRoleFinance" value="${true}"/>
                         </g:hasRoleFinance>
-                        <g:each var="propertyName" in="${command.availableProperties}">
-                            <g:set var="disabled" value="${'pricePerUnit'.equals(propertyName) && !hasRoleFinance}"/>
-                            <option value="${propertyName}"
-                                ${command.selectedProperties?.toList()?.contains(propertyName)?'selected':''}
+                        <g:each var="property" in="${command.availableProperties}">
```

Comment:
I am also wondering if we could go with a different approach and instead of bringing these methods from Grails 1, get a list of `persistentProperties`  and map them to a `messeges.properties` label so we have these option labels translatable.